### DISCUSSION TITLE
Welcome to Robocorp Python library Discussions!

### DISCUSSION THREAD

**Discussion by rihardsgravis** [robocorp/robocorp#137] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by n19k6** [comment]
Hello, my name is Raphael, I work as a test automation engineer for already about 15 years also not always full time.
Currently I switched from perl to python.
Unfortunaly I have do deal with different programming languages at the same time in different project, so sometimes I get confused.
Currently I am using robocorp-windows (python) in an PoC and Flaui (C#) in a different project.
Four years ago I was so upset by a commercial testtool that automated a use case using hardware - that is just a side husle.
If you interested search for "NON-INTRUSIVE TEST EXECUTION OF GUI APPLICATIONS USING OFF-THE-SHELF HARDWARE"
Cheers,
Raphael
